libs/accounts/recovery-phone/src/lib/sms.manager.ts

```diff
@@ -70,12 +70,26 @@ export class SmsManager {
     retryCount: number
   ): Promise<MessageInstance> {
     const from = this.rotateFromNumber();
+
     try {
-      const msg = await this.client.messages.create({
-        to,
-        from,
-        body,
-      });
+      let msg: MessageInstance;
+      if (process.env.NODE_ENV === 'development' || 'dev') {
+        // Mocked response for local development
+        msg = {
+          sid: 'mockSid',
+          status: 'queued',
+          to,
+          from,
+          body,
+        } as MessageInstance;
```

Comment:
@LZoog temporary workaround for local manual testing

Reply:
Turns out this isn't needed if using a valid magic number when setting up a recovery phone
https://www.twilio.com/en-us/blog/twilio-test-credentials-magic-numbers-html